extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/pom.xml

```diff
@@ -94,6 +94,12 @@
             <artifactId>guice</artifactId>
         </dependency>
 
+        <!-- Guice Assisted Inject Extension -->
+        <dependency>
```

Comment:
I need this in order to create `AccessEnforcingDelegatingTunnel` instances. Since the `DelegatingGuacamoleTunnel` requires the wrapped tunnel at creation time, I can't use the same `Provider` + init method approach used elsewhere.